jerry-core/parser/js/byte-code.h

```diff
@@ -617,6 +617,8 @@
  */
 typedef struct
 {
+  uint16_t size;                    /**< real size >> MEM_ALIGNMENT_LOG */
```

Comment:
This could be done in a follow-up patch. This patch is big enough, and I am sure such a change will require several line changes.